docs/lang/articles/gui/ggui.md

```diff
@@ -71,42 +79,48 @@ camera.projection_mode(mode)
 scene.set_camera(camera)
 ```
 
-
 ### Configuring light sources
-#### Adding a point light
+
+#### Add a point light
+
+Call `point_light()` to add a point light to the scene.
+
 ```python
 scene.point_light(pos, color)
 ```
-Note that `point_light` method needs to be called every frame. Similar to the `canvas` methods, you should call this within your render loop.
 
+Note that you need to call `point_light()` for every frame. Similar to the `canvas()` methods, call this method within your render loop.
+
+### 3D Geometries
 
-### 3d Geometries
 ```python
 scene.mesh(vertices, indices, normals, color, per_vertex_color)
 scene.particles(vertices, radius, color, per_vertex_color)
 ```
 
-The arguments `vertices`, `indices`, `per_vertex_color`, and `image` are all expected to be Taichi fields. If `per_vertex_color` is provided, `color` will be ignored.
+The arguments `vertices`, `indices`, `per_vertex_color`, and `image` are all expected to be Taichi fields. If `per_vertex_color` is provided, `color` is ignored.
 
 The positions/centers of geometries should be in the world-space coordinates.
 
 :::note
 
-If a mesh has `num` triangles, the `indices` should be a 1D scalar field with a shape of `num * 3` instead of a vector field.
+If a mesh has `num` triangles, the `indices` should be a 1D scalar field with a shape of `num * 3`, *not* a vector field.
```

Comment:
I'd prefer this to be length instead of shape, it's easier to understand in the 1D case ;)